server/plugin/command.go

```diff
@@ -64,6 +63,12 @@ func (p *Plugin) getAutoCompleteData() *model.AutocompleteData {
 	subscribe := model.NewAutocompleteData("boards subscribe", "", "Add a subscription")
 	azureDevops.AddCommand(subscribe)
 
+	subscriptions := model.NewAutocompleteData("boards subscriptions", "", "View board's subscriptions in the current channel")
```

Comment:
What will be better boards subscriptions or board subscription ?

Reply:
same as above